### ISSUE TITLE
stuck open trades

### ISSUE THREAD

**Issue by marian001** [bisq-network/bisq#2788]
From time to time some trades don't seem to initialize and end up in Open Trades list without any option to delete them...   This means none of these trades succeded even with the first step of Trade process, so no transactions related to these trades have been executed. Each trade is with another buyer, while I'm BTC seller as maker. Two trades use SEPA as payment method, one uses Revolut. All of these trades trade pair BTC/EUR.
I now have 3 such trades stuck in Open Trades...

**Comment by marian001** [comment]
I'd like to correct statement about no transactions. In fact, maker fee transactions were executed and are confirmed on blockchain.

**Comment by ripcurlx** [comment]
Could you please post the last couple of lines of your log file when you are opening the open trades screen? 

> Without any option to delete them

Do you mean you don't see the delete icon in open trades or that it fails with an exception if you click on it?

**Comment by ManfredKarrer** [comment]
@marian001 What is the county of your bank at the sepa transfer? We blocked several countries due the chargeback scammer until the next release where we add more protection tools. See https://bisq.community/t/urgent-certain-banks-banned-until-further-notice/7420/

**Comment by ManfredKarrer** [comment]
Can you open a dispute with cmd+o or ctrl+o?

**Comment by marian001** [comment]
> 
> 
> Could you please post the last couple of lines of your log file when you are opening the open trades screen?


máj-01 01:10:06.533 [JavaFX Application Thread] WARN  b.c.trade.Trade: depositTx is null 
máj-01 01:10:06.533 [JavaFX Application Thread] WARN  b.c.trade.Trade: depositTx is null 
máj-01 01:10:06.533 [JavaFX Application Thread] WARN  b.c.trade.Trade: depositTx is null 
máj-01 01:10:06.533 [JavaFX Application Thread] WARN  b.c.trade.Trade: depositTx is null 
máj-01 01:10:06.534 [JavaFX Application Thread] WARN  b.c.trade.Trade: depositTx is null 
máj-01 01:10:06.534 [JavaFX Application Thread] WARN  b.c.trade.Trade: depositTx is null 
máj-01 01:10:06.534 [JavaFX Application Thread] WARN  b.c.trade.Trade: depositTx is null 
máj-01 01:10:06.534 [JavaFX Application Thread] WARN  b.c.trade.Trade: depositTx is null 
máj-01 01:10:17.838 [JavaFX Application Thread] INFO  b.c.p.p.PriceFeedService: request from provider http://5bmpx76qllutpcyp.onion/ 62.001 sec. after last request 
máj-01 01:10:19.004 [JavaFX Application Thread] INFO  b.c.p.p.PriceFeedService: Received new MarketPrice(currencyCode=EUR, price=4712.39, timestampSec=1556665780085, isExternallyProvidedPrice=true) from provider http://5bmpx76qllutpcyp.onion/ after 1.166 sec. 
máj-01 01:10:30.227 [JavaFX Application Thread] INFO  b.c.d.n.l.n.LiteNodeNetworkService: We received a new message from peer Optional[5ws3aiuy7nczxxaq.onion:9999] and broadcast it to our peers. extBlockId=0000000000000000000e0def4bde69312d45ae7b9066a8c50714f3a4e32847cf:[] 
máj-01 01:10:30.228 [JavaFX Application Thread] INFO  b.c.d.n.l.LiteNode: onNewBlockReceived: block at height 573993, hash=0000000000000000000e0def4bde69312d45ae7b9066a8c50714f3a4e32847cf 
máj-01 01:10:30.228 [JavaFX Application Thread] INFO  b.c.d.n.BsqNode: We received an block with a future block height. We store it as pending and try to apply it at the next block. rawBlock: height/hash=573993/0000000000000000000e0def4bde69312d45ae7b9066a8c50714f3a4e32847cf 
máj-01 01:10:30.965 [JavaFX Application Thread] INFO  b.n.p.n.Connection: We did not send the message because the peer does not support our required capabilities. message=NetworkEnvelope{
     messageVersion=10
}, peer=Optional[aqhtyqkeq6uq45fl.onion:9999], peers supportedCapabilities=[0, 1, 2, 7] 
máj-01 01:10:32.721 [Wallet autosave thread] INFO  o.b.w.WalletFiles: Background saving wallet; last seen block is height 573993, date 2019-04-30T23:09:50Z, hash 0000000000000000000e0def4bde69312d45ae7b9066a8c50714f3a4e32847cf 
máj-01 01:10:32.721 [Wallet autosave thread] INFO  o.b.w.WalletFiles: Background saving wallet; last seen block is height 573993, date 2019-04-30T23:09:50Z, hash 0000000000000000000e0def4bde69312d45ae7b9066a8c50714f3a4e32847cf 
máj-01 01:10:32.759 [Wallet autosave thread] INFO  o.b.w.WalletFiles: Save completed in 29.00 ms 
máj-01 01:10:32.764 [Wallet autosave thread] INFO  o.b.w.WalletFiles: Save completed in 33.16 ms 

> 
> > Without any option to delete them
> 
> Do you mean you don't see the delete icon in open trades or that it fails with an exception if you click on it?

Well, Bisq offers delete icon on My Open Offers screen. But not on Open Trades screen. I guess that's normal, but what I meant was that these trades are stuck somewhere at the beginning and I have no way to deal with them.

**Comment by marian001** [comment]
> 
> 
> @marian001 What is the county of your bank at the sepa transfer? We blocked several countries due the chargeback scammer until the next release where we add more protection tools. See https://bisq.community/t/urgent-certain-banks-banned-until-further-notice/7420/

I am aware of German bank accounts being blocked at the moment. All of the SEPA stuck transactions are related to my German bank account. However, there is also one transaction associated with my Revolut account.

**Comment by marian001** [comment]
> 
> 
> Can you open a dispute with cmd+o or ctrl+o?

No, I can't because I can't find related transaction IDs in the dropdown list of deposit transactions. In all of these cases only Maker fee transaction has been made, but IDs of these transactions are not in the list, so I can't proceed with submitting a ticket.

**Comment by ManfredKarrer** [comment]
Just select a random one. There is none created so it does not matter.

**Comment by marian001** [comment]
> 
> 
> Just select a random one. There is none created so it does not matter.

OK, I opened 4 support tickets.

**Comment by nacknacknock** [comment]
The problem mentioned has been reported by multiple users, including myself (https://bisq.community/t/trade-stuck-before-step-1/7245/33). But not for all of them it is possible to even open any support ticket.
Therefore I filed another issue https://github.com/bisq-network/bisq/issues/2795, which I strongly assume constitutes a bug.

**Comment by ManfredKarrer** [comment]
Please update to new release. Should be fixed there.
